runtime/cudaq/platform/default/rest_server/helpers/JsonConvert.h

```diff
@@ -9,8 +9,8 @@
 #pragma once
 #include "common/ExecutionContext.h"
 #include "common/FmtCore.h"
+#include "cudaq/Support/Version.h"
 #include "nlohmann/json.hpp"
-
```

Comment:
Did clang-format remove this blank line?

I think it should be kept as it separates the list of includes from the doxygen comment.

Reply:
I think I might have put the new header at the empty line and clang-format shuffled it. I've added the blank line back as suggested.